libbeat/scripts/Makefile

```diff
@@ -36,7 +36,8 @@ COVERAGE_TOOL?=${BEAT_GOPATH}/bin/gotestcover
 COVERAGE_TOOL_REPO?=github.com/elastic/beats/vendor/github.com/pierrre/gotestcover
 TESTIFY_TOOL_REPO?=github.com/elastic/beats/vendor/github.com/stretchr/testify
 LIBCOMPOSE_TOOL_REPO?=github.com/docker/libcompose
-GOBUILD_FLAGS?=-i
+NOW=$(shell date -u '+%Y-%m-%dT%H:%M:%S')
```

Comment:
This looks to be very close to RFC3339. Can you add the Zulu timezone identifier (e.g. `date -u +"%Y-%m-%dT%H:%M:%SZ"`). This will make the value parse nicely like `time.Parse(time.RFC3339, "2018-01-03T16:03:28Z")`.